cypress/e2e/PageObjects/EIContactUsPo.cy.js

```diff
@@ -0,0 +1,115 @@
+/// <reference types="cypress" />
+import ContactUsEIdata from '../../fixtures/ContactUsEIdata.json'
+
+function EIContactUsHeader() {
+  return cy.get('[data-cy ="eIContactUsContent"]>h1')
+}
+
+function ValidateEIContactUsUrl() {
+  cy.url().should('contains', '/contact-employment-insurance')
+}
+
+function ValidateEIContactUsHeaderEN() {
+  EIContactUsHeader()
+    .should('be.visible')
+    .and('have.text', 'Contact Employment Insurance')
+}
+
+function ValidateEIContactUsUrlFR() {
+  cy.url().should('contains', '/fr/contact-employment-insurance')
+}
+
+function ValidateEIContactUsHeaderFR() {
+  EIContactUsHeader()
+    .should('be.visible')
+    .and('have.text', "Communiquer avec l'assurance-emploi")
+}
+
+function ValidateOnthisPageLinks() {
+  return cy
+    .get('[data-cy ="eIContactUsContent"]>section')
+    .find('a')
+    .should('be.visible')
+    .and('have.length', '4')
+    .and('not.have.length', 0)
+    .and('not.have.attr', 'href', '#undefined')
+}
+
+function ValidateTelephoneLink() {
+  cy.get('[data-cy="tableLink1"]')
+    .should('have.attr', 'target', '_self')
+    .click()
+  cy.url().should('contains', 'ei-contact-telephone')
+}
+
+function ValidateTelephoneHeading() {
+  cy.get('[name="ei-contact-telephone"]>h2').should('have.text', 'Telephone')
+}
+
+function ValidateTelephoneSectionContent() {
+  return cy.get('[name ="ei-contact-telephone"]>div').should('be.visible')
+}
+
+function ValidateCallBackHeading() {
+  cy.get('[name="ei-contact-callback"]>h2').should('have.text', 'Callback')
+}
+
+function ValidateCallbackSectionContent() {
+  return cy.get('[name ="ei-contact-callback"]>div').should('be.visible')
+}
+
+function ValidateEachtableOfContentLink() {
+  for (let i = 0; i < ContactUsEIdata.length; i++) {
+    const LinkName = ContactUsEIdata[i].Link
```

Comment:
LinkName is never used. If its not a typo, we can remove it.

Reply:
> LinkName is never used. If its not a typo, we can remove it.

Yes, I forgot to delete this step, Thanks! It's rmeoved now